spec/requests/unauthorized_api_access_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'POST /api/v1/booking_requests' do
+  scenario 'unauthorized access' do
+    given_a_permitted_gds_sso_user
```

Comment:
I'm a little confused by your intention here, what has this user been "permitted to" given an unauthorised request is then made.